### PR TITLE
Use log4j.xml as default log4j configuration file

### PR DESCRIPTION
**Pull request by byjiang1996**
Test:
Can run azkaban-web-server, azkaban-exec-server and azkaban-solo-server successfully with correct log lines.

### CODE REVIEW

File: azkaban-exec-server/src/main/bash/internal/internal-start-executor.sh [hunk 1]
```diff
@@ -49,10 +49,12 @@ if [[ -z "$AZKABAN_OPTS" ]]; then
...
 fi
 # Set the log4j configuration file
-if [ -f $conf/log4j.properties ]; then
+if [ -f $conf/log4j.xml ]; then
+  AZKABAN_OPTS="$AZKABAN_OPTS -Dlog4j.configuration=file:$conf/log4j.xml -Dlog4j.log.dir=$azkaban_dir/logs"
+elif [ -f $conf/log4j.properties ]; then
   AZKABAN_OPTS="$AZKABAN_OPTS -Dlog4j.configuration=file:$conf/log4j.properties -Dlog4j.log.dir=$azkaban_dir/logs"
 else
-  echo "Exit with error: $conf/log4j.properties file doesn't exist."
+  echo "Exit with error: neither $conf/log4j.xml or $conf/log4j.properties exist."
```
Comment (djaiswal83) [inline azkaban-exec-server/src/main/bash/internal/internal-start-executor.sh:57]: nor

File: azkaban-web-server/src/main/bash/internal/internal-start-web.sh [hunk 2]
```diff
@@ -48,10 +48,12 @@ if [[ -z "$AZKABAN_OPTS" ]]; then
...
 fi
 # Set the log4j configuration file
-if [ -f $conf/log4j.properties ]; then
+if [ -f $conf/log4j.xml ]; then
+  AZKABAN_OPTS="$AZKABAN_OPTS -Dlog4j.configuration=file:$conf/log4j.xml -Dlog4j.log.dir=$azkaban_dir/logs"
+elif [ -f $conf/log4j.properties ]; then
   AZKABAN_OPTS="$AZKABAN_OPTS -Dlog4j.configuration=file:$conf/log4j.properties -Dlog4j.log.dir=$azkaban_dir/logs"
 else
-  echo "Exit with error: $conf/log4j.properties file doesn't exist."
+  echo "Exit with error: neither $conf/log4j.xml or $conf/log4j.properties exist."
```
Comment (djaiswal83) [inline azkaban-web-server/src/main/bash/internal/internal-start-web.sh:56]: nor

File: azkaban-exec-server/src/main/resources/conf/azkaban.properties [hunk 3]
```diff
@@ -42,11 +42,12 @@ azkaban.jobtype.plugin.dir=plugins/jobtypes
 database.type=mysql
 mysql.port=3306
 mysql.host=localhost
-mysql.database=azkaban
-mysql.user=azkaban
+mysql.database=az_test
```
Comment (djaiswal83) [inline azkaban-exec-server/src/main/resources/conf/azkaban.properties:45]: Why do we need this change?

**Review by djaiswal83 — CHANGES_REQUESTED** [review]
Thanks for making the change. I have few comments.

File: azkaban-exec-server/src/main/resources/conf/azkaban.properties (hunk 3 above)
Comment (byjiang1996) [reply azkaban-exec-server/src/main/resources/conf/azkaban.properties]: The changes here are aborted.

File: azkaban-exec-server/src/main/bash/internal/internal-start-executor.sh [hunk 4]
```diff
@@ -54,7 +54,7 @@ if [ -f $conf/log4j.xml ]; then
 elif [ -f $conf/log4j.properties ]; then
   AZKABAN_OPTS="$AZKABAN_OPTS -Dlog4j.configuration=file:$conf/log4j.properties -Dlog4j.log.dir=$azkaban_dir/logs"
 else
-  echo "Exit with error: neither $conf/log4j.xml or $conf/log4j.properties exist."
```
[the remark this answers is not in the document]
Comment (byjiang1996) [reply azkaban-exec-server/src/main/bash/internal/internal-start-executor.sh]: Done... Sorry about that

**Review by djaiswal83 — APPROVED** [review]
Looks good to me.

**Review by hqin2016 — APPROVED** [review]
LGTM

**Pull request merged**